src/parquet/file/writer-internal.cc

```diff
@@ -141,7 +142,7 @@ ColumnWriter* RowGroupSerializer::NextColumn() {
   col_meta->meta_data.__set_type(ToThrift(column_descr->physical_type()));
   col_meta->meta_data.__set_path_in_schema(column_descr->path()->ToDotVector());
   std::unique_ptr<PageWriter> pager(
-      new SerializedPageWriter(sink_, Compression::UNCOMPRESSED, col_meta, allocator_));
+      new SerializedPageWriter(sink_, properties_->compression(), col_meta, allocator_));
```

Comment:
When we have also support for defining compression per column, we should use a function of properties that gets the column path and then returns either the specific codec if one is defined for the column, otherwise the default.